polyaxon/requirements.yaml

```diff
@@ -1,17 +1,17 @@
 dependencies:
   - name: postgresql
-    version: 0.8.7
```

Comment:
I think this change will have to be made at v1.0, bit for v0.x this will be a huge breaking change for any running cluster.

Reply:
reverted dependency on PostgreSQL